app/lib/submission_builder/ty2022/states/ny/documents/state1099_g.rb

```diff
@@ -29,6 +29,12 @@ def document
                 end
                 xml.RecipientSSN recipient.ssn
                 xml.RecipientName recipient.full_name
+                xml.RecipientUSAddress do
+                  xml.AddressLine1Txt form1099g.recipient_street_address
```

Comment:
is there a AddressLine2? just curious

Reply:
Yes for the Apt. But we don't have a separate field in the database for apt. I think the user would put it in the same line as the street address